.travis.yml

```diff
@@ -8,22 +9,43 @@ env:
   - BRANCH=0.19.6
   - BRANCH=0.20.2
   # This is the latest working Nim version against which Nimble is being tested
+  # Update in cache:directories as well
   - BRANCH=#2565d3d102efd21ba02ed1f3b96d892fe2637d2b
 
 cache:
   directories:
+    - "$HOME/.nimble/bin"
     - "$HOME/.choosenim/toolchains/nim-0.19.6"
     - "$HOME/.choosenim/toolchains/nim-0.20.2"
+    - "$HOME/.choosenim/toolchains/nim-#2565d3d102efd21ba02ed1f3b96d892fe2637d2b"
```

Comment:
Wouldn't a simple:

```
  - "$HOME/.choosenim/toolchains/nim-$BRANCH"
```

Work here?

Reply:
If only :( vars defined in the yaml don't work there.